### PR TITLE
[tvos][videotoolbox] Enable VideoToolbox.framework on tvOS

### PR DESCRIPTION
**Pull request by spouliot**
This is just a build enablement to see xtro results. API adjustments,
including availability attributes, remains to be done.

### CODE REVIEW

File: src/VideoToolbox/VTFrameSilo.cs [hunk 1]
```diff
@@ -76,7 +76,7 @@ public static VTFrameSilo Create (NSUrl fileUrl = null, CMTimeRange? timeRange =
 			var status = VTFrameSiloCreate (
 				IntPtr.Zero,
 				fileUrl == null ? IntPtr.Zero : fileUrl.Handle, 
-				timeRange ?? CMTimeRange.Invalid, 
+				timeRange ?? CMTimeRange.InvalidRange,
```
Comment (dalexsoto) [inline src/VideoToolbox/VTFrameSilo.cs]: great catch 👍

File: src/VideoToolbox/VTFrameSilo.cs [hunk 2]
```diff
@@ -154,7 +154,7 @@ static VTStatus BufferCallback (IntPtr callbackInfo, IntPtr sampleBufferPtr)
 		public VTStatus ForEach (Func<CMSampleBuffer, VTStatus> callback, CMTimeRange? range = null)
 		{
 			callbackHandle = GCHandle.Alloc (callback);
-			var foreachResult = VTFrameSiloCallFunctionForEachSampleBuffer (handle, range ?? CMTimeRange.Invalid, GCHandle.ToIntPtr (callbackHandle), static_EachSampleBufferCallback);
+			var foreachResult = VTFrameSiloCallFunctionForEachSampleBuffer (handle, range ?? CMTimeRange.InvalidRange, GCHandle.ToIntPtr (callbackHandle), static_EachSampleBufferCallback);
```
Comment (dalexsoto) [inline src/VideoToolbox/VTFrameSilo.cs]: great catch 👍

File: src/VideoToolbox/VTMultiPassStorage.cs [hunk 3]
```diff
@@ -88,7 +88,7 @@ protected virtual void Dispose (bool disposing)
 			var status = VTMultiPassStorageCreate (
 				IntPtr.Zero,
 				fileUrl == null ? IntPtr.Zero : fileUrl.Handle, 
-				timeRange ?? CMTimeRange.Invalid, 
+				timeRange ?? CMTimeRange.InvalidRange,
```
Comment (dalexsoto) [inline src/VideoToolbox/VTMultiPassStorage.cs]: great catch 👍

**Comment by spouliot** [comment]
introspection - tvOS (Failed) - maybe the test needs adjustment ? because the API should be available. Needs some investigation before merging.

```
[FAIL] ApiCMAttachmentTest.CheckFailAttachments : System.InvalidOperationException : Could not create the new instance for type VTCompressionSession.
```

known failures:
* monotouch-test crash -> https://bugzilla.xamarin.com/show_bug.cgi?id=52162

**Comment by spouliot** [comment]
known failure:
* monotouch-test - Unified 32bits -> https://bugzilla.xamarin.com/show_bug.cgi?id=52162

**Pull request merged**